docs/blog/newsletter-june-2023.rst

```diff
@@ -0,0 +1,128 @@
+
+The practice of adding comments also has benefits. Given that developers are more likely to read the code samples, comments can be a good way to make sure developers see information they really need to use the samples. Also, because code samples are often designed to be copied and pasted, code comments help ensure that important information stays with the code no matter where it is used.
+
+A related question is where to describe placeholders, those brief bits of text in code samples that users must replace with their own values before deploying the code. Placeholder descriptions are typically included in the explanatory text for a code sample rather than within the sample itself.
+
+To read more about comments and code samples, consider these two books: |book_one|_ and |book_two|_.
+
+.. _book_one: https://www.pearson.com/en-us/subject-catalog/p/developing-quality-technical-information-a-handbook-for-writers-and-editors/P200000000159/9780133118971
+.. |book_one| replace:: *Developing Quality Technical Information: A Handbook for Writers and Editors*
+
+.. _book_two: https://docsfordevelopers.com/
+.. |book_two| replace:: *Docs for Developers: An Engineer’s Field Guide to Technical Writing*
+
+-----------------------
+Extreme take-home tests
+-----------------------
+
+With the current spate of layoffs and recent graduations, interviewing practices are hot topics. In a recent discussion, a job hunter asked about the appropriateness of spending a lot of time on a take-home writing test about a company’s product. Working on a take-home test presents certain issues to be aware of.
+
+^^^^
+Time
+^^^^
+
+The general consensus was to limit any time spent to about 1 hour. Some indicated willingness to spend more time. One person indicated spening an hour doing the assignment, but then providing "an explanation of why I wasn't going to respond to all of their requests."
+
+^^^^^
+Money
+^^^^^
+
+Many in the thread cautioned about doing work about a company’s product for free. One indicated that expecting significant unpaid work was "a big red flag." This can be a way for a company to get free work from a lot of people. Several remarked that they’ve received payment for completing take-home tests. As an interviewee, you may need to ask about payment, especially if the test requires significant work.
+
+^^^^^^^^^
+Ownership
+^^^^^^^^^
+
+If you do a writing test, does it belong to you or to the company? In general, if you’re writing about a company’s product, it would belong to the company. You may want to get this clarified because it may affect how long you work on the test and whether you have rights to the work itself. If you spend a lot of time on a writing sample, it may work as a good portfolio piece. So it's worth reesolving the question.
```

Comment:
```suggestion
If you do a writing test, does it belong to you or to the company? In general, if you’re writing about a company’s product, it would belong to the company. You may want to get this clarified because it may affect how long you work on the test and whether you have rights to the work itself. If you spend a lot of time on a writing sample, it may work as a good portfolio piece. So it's worth resolving the question.
```